### DISCUSSION TITLE
请问怎么去掉登陆首页那个蓝方块爬梯的动画

### DISCUSSION THREAD

**Discussion by bulb888** [AlistGo/alist#1526] [Q&A]
如题~小白
![P%%(L@SANFV8C2XNRP515T](https://user-images.githubusercontent.com/110551961/185528566-4dfc3f6a-0775-4ef0-ab3d-ed9c145c4811.png)

**Comment by Pandaft** [comment]
奇怪的需求增加了（

**Comment by anwen-anyi** [comment]
> 如题~小白

后台不是有个动画按钮吗…关了就行

**Reply by bulb888** [reply]
这个是吗？这个不是吧
